plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTablePqlExtractor.java

```diff
@@ -92,7 +100,11 @@ public static String extractPql(DynamicTable table, TupleDomain<ColumnHandle> tu
             }
             builder.append(table.limit().getAsLong());
         }
-        return builder.toString();
+        String pql = builder.toString();
+        if (queryLoggingEnabled && log.isInfoEnabled()) {
+            log.info("Generated PQL: %s", pql);
```

Comment:
Please revert config changes. We can control log level with [log.properties](https://trino.io/docs/current/installation/deployment.html#log-levels).

Also, please change to debug level. Same for another place.